docs/en/stack/data-frames/dataframe-examples.asciidoc

```diff
@@ -0,0 +1,161 @@
+[role="xpack"]
+[testenv="basic"]
+[[dataframe-examples]]
+== {dataframe-cap} examples
+++++
+<titleabbrev>Examples</titleabbrev>
+++++
+
+beta[]
+
+This page provides examples of how to use {dataframe-transforms} to derive useful 
+insights from your data. For a more detailed, step-by-step example, see 
+<<ecommerce-dataframes,Transforming your data with {dataframes}>>.
+
+* <<example-clientips>>
+
+
+[float]
+[[example-clientips]]
+=== Finding suspicious client IPs by using scripted metrics
+
+With {dataframe-transforms}, you can use 
+{ref}/search-aggregations-metrics-scripted-metric-aggregation.html[scripted 
+metric aggregations] on your data. These aggregations are flexible and make 
+it possible to perform very complex processing. Let's use scripted metrics to 
+identify suspicious client IPs in the web log 
+{kibana-ref}/add-sample-data.html[{kib} sample dataset].
+
+The example below transforms web log data into an entity-centric index where the 
```

Comment:
In my opinion, this introductory sentence doesn't give me too much context of what I'm about to see in the example. If we can summarize what the transform is going to do, I think that's more helpful.  Something like:

For example, let's transform the data such that the new index contains the sum of bytes, the number of distinct URLs, and the number of distinct geographic destinations for each client IP. We'll also use a scripted field to count the specific types of HTTP responses that each client IP receives.

Reply:
You're right. That's why I added the "Based on..." phrases after the questions at the bottom of the doc, but it is a much better solution. I've tried to keep the example relatively short in terms of prose because the API request and the response are fairly long. However, my intention of keeping it short would hurt the overall lucidity of the example. I amended accordingly.